### PR TITLE
add vearch to Llama_index

### PR DESCRIPTION
**Pull request by zhanghexian**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new notebook (that tests end-to-end)

# Suggested Checklist:

- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: docs/examples/vector_stores/VearchDemo.ipynb [hunk 1]
```diff
@@ -0,0 +1,194 @@
+{
```
Comment (hatianzhang) [inline docs/examples/vector_stores/VearchDemo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    集群版
</pre><p>could you change this to english</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (hatianzhang) [inline docs/examples/vector_stores/VearchDemo.ipynb:1]: <p>why the response mixed with Chinese tokens?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (zhanghexian) [reply docs/examples/vector_stores/VearchDemo.ipynb:1]: fixed,thanks!

**Comment by zhanghexian** [comment]
now i had fix the problem,please check,thanks!

**Comment by hatianzhang** [comment]
@zhanghexian thanks! but seems there are some unit tests failed. could you help resolve

**Comment by zhanghexian** [comment]
I am sorry, now I fix this problem,but vearch packet is exist i don't know why it can't import,so i delete it.please check,thanks!

File: llama_index/vector_stores/vearchdb.py [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+            None
+        """
+        if len(ref_doc_id) == 0:
+            return
+        ids: List[str] = []
+        ids.append(ref_doc_id)
+        self._delete(ids)
+
+
+
```
Comment (hatianzhang) [inline llama_index/vector_stores/vearchdb.py:467]: nit remove those lines

**Comment by zhanghexian** [comment]
now removed,thanks!

**Comment by zhanghexian** [comment]
can you tell me how to make lint? i check it based CONTRIBUTE.MD，but  it show me "make: *** 没有规则可以创建目标“lint”, thanks!

**Comment by hatianzhang** [comment]
@zhanghexian you could install `pip install pre-commit`

**Comment by logan-markewich** [comment]
@zhanghexian apologies, with v0.10.0 the project structure has changed slightly

I can take a stab at porting this PR in a bit, if you don't get to it first :)

**Pull request closed without merging**